share/spice/zomato/zomato.css

```diff
@@ -0,0 +1,12 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/zomato/zomato.css
+    Created on : 2015-07-30 09:08 AM
+    Author     : dheeraj143
+    Description:
+    Purpose of the stylesheet follows.
+    To change this template use Tools | Templates.
+*/
+
+.tile--loc .tile__media__img {
```

Comment:
Please namespace the CSS by prefacing the css selector with `.tile__zomato` or `.zci__zomato` so that this only affects your Spice's tiles